.venv/bin/keylistattack.py

```
@@ -0,0 +1,271 @@
+            logging.warning("Flag -full will have no effect")
+        if options.t is not None:
+            targets.append(options.t)
+        elif options.tf is not None:
+            try:
+                with open(options.tf, 'r') as f:
+                    for line in f:
+                        target = line.strip()
+                        if target != '' and target[0] != '#':
+                            targets.append(target + ":" + "N/A")
+            except IOError as error:
+                logging.error("Could not open file: %s - %s", options.tf, str(error))
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)